config/samples/registry_v1alpha1_devfileregistry.yaml

```diff
@@ -3,5 +3,5 @@ kind: DevfileRegistry
 metadata:
   name: devfileregistry-sample
 spec:
-  # Add fields here
-  foo: bar
+  bootstrapImage: quay.io/devfile/metadata-server:latest
```

Comment:
We may need to do the following changes: 
`bootstrapImage` -> `devfileIndexImage`
`metadata-server` -> `index-server`

Reply:
Ah, missed that file. It's just a sample custom resource file generated by `operator-sdk`. It's not currently used by anything. But, it should be updated for reference.

Reply:
Updated.